openedx/core/djangoapps/user_api/tests/test_views.py

```diff
@@ -861,7 +850,7 @@ def test_register_form_default_fields(self):
                 u"name": u"username",
                 u"type": u"text",
                 u"required": True,
-                u"label": u"Username",
+                u"label": u"Public username",
```

Comment:
We were explicitly asked to change to 'Public username' as per the updated comps